spec/board_spec.rb

```diff
@@ -5,67 +5,75 @@
 RSpec.describe Board do
   let(:board) {Board.new}
 
-  it "should return all the position" do
-    expect(board.board).to eq([[0, 1, 2], [3, 4, 5], [6, 7, 8]])
-  end
-
-  it "should be free position when it never been set" do
+  it "should be a free position when position free" do
     position = 0
+    board.string_to_board("   ,   ,   ")
     expect(board.free_positions.include?(position)).to be true
   end
 
-  it "should shouldn't get the spot after it set" do
+  it "shouldn't be a free position when postion isn't free" do
```

Comment:
Same for the test before "should be a free position when position free". That doesn't say anything about what is actually expected to happen.

Additionally to that using "should" in test descriptions has become an anti-pattern. It's too lose in the sense that "should" can imply that something  can happen but it's also okay if it doesn't. But in our tests we want to make sure that an expectation is met. The test name should reflect that.